week_3/2018202196ZYL/src/crawler.cpp

```diff
@@ -0,0 +1,50 @@
+#include<iostream>
+#include<cstring>
+#include<string>
+#include<regex>
+#include<fstream>
+#include<sstream>
+#include<ios>
+#include<set>
+#include<queue>
+#include"crawler.h"
+#include"total.h"
+
+void crawler::bfs()
+{
+	while(!str_queue.empty())
+	{
+		std::string wget_str;
+		std::string str;
+		ans++;
+		//const char* str_que=str_queue.front().data();
+		//sprintf(wget_str,"wget %s -O ./home/%d.html -T 20 -q",str_que,ans);
+		wget_str="wget \""+str_queue.front()+"\" --output-document=./home/"+std::to_string(ans)+".html -t 3 -T 15 -q";
+		system(wget_str.c_str());
+		std::cout<<ans<<' '<<str_queue.front()<<std::endl;
+		str_queue.pop();
+		//sprintf(str,"./home/%d.html",ans);
+		str="./home/"+std::to_string(ans)+".html";
+		std::ifstream cur_url (str.c_str());

```

Comment:
Indent your code with either spaces or tabs, not BOTH spaces AND tabs. Tabs' length may be different in everywhere.